modules/identity-provider-oidc-CR.adoc

```diff
@@ -43,9 +43,7 @@ identity name. It is also used to build the redirect URL.
 client must be allowed to redirect to
 `\https://oauth-openshift.apps.<cluster_name>.<cluster_domain>/oauth2callback/<idp_provider_name>`.
 <4> Reference to an {product-title} `Secret` object containing the client secret.
-<5> List of claims to use as the identity. First non-empty claim is used. At
-least one claim is required. If none of the listed claims have a value,
-authentication fails. For example, this uses the value of the `sub` claim in the returned `id_token` as the user's identity.
+<5> The list of claims to use as the identity. The first non-empty claim is used.
```

Comment:
The change is fine.
However, searching `sub`, the doc page's other places still has:
```
At least one claim must be configured to use as the user’s identity. The standard identity claim is sub.

sub | Short for "subject identifier." The remote identity for the user at the issuer.
```
It is better to clarify to end users that in v4 user's identity can't be configured, using `sub` as user's identity is hard coded in v4 (you can elaborate the wording)

Reply:
@xingxingxia Thanks. So we do still have "sub" in that table even in the newest version of the docs: https://docs.openshift.com/container-platform/4.11/authentication/identity_providers/configuring-oidc-identity-provider.html

Are you saying that we should remove that "sub" entry from the table too?

Reply:
Sorry I did not said clearly. Need not remove `sub` entry from the table. I mean v4 uses `sub` claim as user’s identity in hard code manner. So `At least one claim must be configured to use as the user’s identity` is not correct in v4, because we can configure for user's identity.

Reply:
In terms of previous comment, `as the identity` is also not precise, the claims are used for user info like name, email etc but not for identity.
```suggestion
<5> The list of claims to use as the user various info. The first non-empty claim is used.
```